README.md

```diff
@@ -1,2 +1,133 @@
-# IoT-ChatBot
-Open Projects 2021
+# ***IoTChatbot***
+![image](https://user-images.githubusercontent.com/76246968/127762969-0b0b08ea-84e5-41cd-a843-ae954c220526.png)
```

Comment:
Center align all the images using the `<p>` paragraph tag and `align` attribute. For eg:
`<p align="center">`
  `<img src="https://github.com........." alt="LED Matrix">`
  `<i>LED Matrix</i>`
`</p>`

Refer this README for the same: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

Also resize a few images by assigning the dimensions.